### DISCUSSION TITLE
네이버맵에서 에러가 뜹니다

### DISCUSSION THREAD

**Discussion by silverdust8** [changdoc/naver-vaccine-macro#272] [Q&A]
네이버맵으로 돌릴떄 
console란에서 확인을 해보니
Uncaught (in promise) Error: Network error: Timeout exceeded 라는 말이 계속 뜨고있네요
이러면 안되는건가요?

**Comment by changdoc** [comment]
네이버 서버가 응답이 없는 상태에서 찍히는 에러라 반복해서 찍히고 있지 않다면 괜찮을 것 같습니다.

예상하기로는 질병 관리청에서 응답이 지연되면서 발생하는 에러 일 것 같네요.
